composer.json

```diff
@@ -1,22 +1,39 @@
 {
     "name": "site5/phantoman",
     "description": "The Codeception extension for automatically starting and stopping PhantomJS when running tests.",
-    "keywords": ["codeception", "extension", "phantomjs", "headless"],
+    "keywords": [
+        "codeception",
+        "extension",
+        "phantomjs",
+        "headless"
+    ],
     "homepage": "https://github.com/grantlucas/phantoman",
     "license": "MIT",
     "authors": [
         {
             "name": "Grant Lucas",
             "email": "contact@grantlucas.com"
+        },
+        {
+            "name": "Jonas Hoppe",
+            "email": "pwnyprod@gmail.com"
         }
     ],
+    "scripts": {
+        "tests": "vendor/bin/codecept build && vendor/bin/codecept run"
+    },
     "require": {
-        "php": ">=5.4.0",
-        "codeception/codeception": "^2.2.7"
+        "php": ">=7.4",
```

Comment:
Do any changes in this PR _restrict_ the usage of any PHP version below PHP 7.4? This package is used by a lot of systems running on PHP Versions lower than 7.4. Codeception's own PHP version requirements are `"php": ">=5.6.0 <8.0"` which I'd be inclined to match or drop down to 7.2+ if possible so that we can at least get good type hinting and returns etc.

Reply:
by using typehints for all methods and properties we need 7.4. We can lower to 7.2 by not using property typehints.

Reply:
Thanks. I think, in order to keep this extension as accessible as possible, it'd be good to be able to drop down to 7.2. There are still a large number of people on older PHP versions and since Codeception itself doesn't have a 7.4 requirement, adding that requirement here would prevent Phantoman usage when, in theory, it could have been possible.

My goal with this project is to try to keep it as backwards compatible as possible. If you can believe it, in Oct **2020* Version 2.x of this package _finally_ passed version 1.x. 2.x was released in 2017.

With the changes happening already, we'll likely have to bump major version numbers to 3.x which will limit some usage already. If we can support back to 7.2, that'll remove a major hurdle from this package being used by those using Codeception but maybe not the latest PHP version.

Thanks

Reply:
Ok, i will find time to downgrade the Version to php7.2 and i think it is okay while php7.1 is nearly EOL, so normaly the maintained Projects shouldnt use a lower Version anymore 🤣 (We all know these things never happen).

Anyways this is fine for me so if i have time i will update the PR and solve the Comments ;)

Reply:
PS: Thanks for the great feedback and the conversation. So the next Step is make it compatible with composer2? :D

Reply:
Thanks for looking into 7.2 support! Much appreciated. Yeah I know 7.1 shouldn't be around much anymore but we all know how that works 😄 so 7.2's a compromise for now for most support but not _too_ far back.